src/core/InstrumentFunctions.cpp

```diff
@@ -409,7 +409,7 @@ void InstrumentFunctionArpeggio::processNote( NotePlayHandle * _n )
 		// Skip notes randomly
 		if( m_arpSkipModel.value() )
 		{
-			if( 100 * ( (float) rand() / (float)( RAND_MAX + 1.0f ) ) < m_arpSkipModel.value() )
+			if( 100 * ( (float) rand() / (static_cast<float>(RAND_MAX) + 1.0f)) < m_arpSkipModel.value() )
```

Comment:
Well since we're on the line...
```suggestion
			if (100 * (static_cast<float>(rand()) / (static_cast<float>(RAND_MAX) + 1.0f)) < m_arpSkipModel.value())
```